awspub/configmodels.py

```diff
@@ -94,6 +95,31 @@ class ConfigImageSSMParameterModel(BaseModel):
     )
 
 
+class ConfigSNSNotificationProtocol(str, Enum):
+    default = "default"
+    email = "email"
+    email_json = "email-json"
```

Comment:
have you tried all of those methods? How do you eg. define a `email-json` message? does it work to have the message as a string and then our `json.dumps` handles that correctly? 
I wonder if we should just restrict this for now to only allow the `default` key until we tested more of that.

Reply:
I agree with allowing `default` key only for now. I tested with `email-json` and `lambda` but couldn't test all the cases. The `email-json` configuration should have json string type message. That can be something like:

```
email-json: "{/"key1/": /"value/"}"
```

This is because our config only accepts strings, otherwise this should be changed with using `json.dump()` if message has dicitionary-like value.